source/beamline/REIXS/REIXSBeamline.cpp

```diff
@@ -97,6 +97,9 @@ REIXSBeamline::REIXSBeamline() :
 	tfyDetector_ = new CLSBasicScalerChannelDetector("TFY", "TFY", scaler_, 4, this);
 	pfyDetector_ = new CLSBasicScalerChannelDetector("PFY", "PFY", scaler_, 3, this);
 
+	I0Current_ = new AMReadOnlyPVControl("I0Current","BL1610-ID-2:mcs16:fbk", this, "I0 Current");
```

Comment:
Not the biggest deal, but member variables should start with lower case (i.e.: `i0Current_` and `teyCurren_`).